tensorboard/uploader/uploader.py

```diff
@@ -833,3 +836,35 @@ def _varint_cost(n):
         result += 1
         n >>= 7
     return result
+
+
+def _filter_graph_defs(events):
+    for e in events:
+        for v in e.summary.value:
+            if (
+                v.metadata.plugin_data.plugin_name
+                != graphs_metadata.PLUGIN_NAME
+            ):
+                continue
+            if v.tag == graphs_metadata.RUN_GRAPH_NAME:
```

Comment:
This seems unnecessarily restrictive.  In practice, for now, we have only run-level graphs anyway.  But IIRC we expect to allow `"__op_graph__/foo"` and `"__conceptual_graph__/bar"` also, and those should also be filtered.  NBD for now but maybe add a comment or TODO to address this later.

(I know this is a straight refactor of code that already ignored those cases, but still)

Reply:
Oh right, I forgot about the different plugin names.  So this condition is protecting us from data that is associated with the graph plugin, but under a different tag.  There is currently no such thing; if something arises in the future, we can't predict whether it should be filtered or not-- so now I agree the condition is correct.